### PR TITLE
(2.9.x) DDF-2601 Fixed NPE in AbstractCswSource.java

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
This PR removes the potential for an NPE in AbstractCswSource when creating the sort order.  This was observed when using SortBy.NATURAL_ORDERING.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@rzwiefel @lcrosenbu 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Data](https://github.com/orgs/codice/teams/data)
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@kcwire
#### How should this be tested? (List steps with links to updated documentation)
Build / Install / Configure Federated CSW / Observe No NPE
#### What are the relevant tickets?
[DDF-2601](https://codice.atlassian.net/browse/DDF-2601)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [X] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/TestCswSource.java [hunk 1]
```diff
@@ -278,6 +278,56 @@ public void testPropertyIsLikeQuery()
...
+        } catch (CswException e) {
+            fail("Could not configure Mock Remote CSW: " + e.getMessage());
+        }
+
+        QueryImpl query = new QueryImpl(builder.attribute(Metacard.ANY_TEXT)
+                .is()
+                .like()
+                .text(searchPhrase));
+        query.setPageSize(pageSize);
+        query.setSortBy(SortBy.NATURAL_ORDER);
```
Comment (shaundmorris) [inline catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/TestCswSource.java]: @bdeining I'm probably just confused here, but should this sortBy be set to `null` for this test?

Comment (bdeining) [inline catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/TestCswSource.java]: However, the unit test name is misleading and should be changed.  Additionally, there should be a test for when `sortBy` is null

Comment (shaundmorris) [inline catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/TestCswSource.java]: sounds good, thanks @bdeining

Comment (bdeining) [inline catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/TestCswSource.java]: @shaundmorris updated unit tests

**Pull request merged**